backend/main.py

```
@@ -522,3 +526,12 @@ def buy_item(
     }
 
 # uvicorn main:app --reload
+
+app.mount(
+    "/static",
+    StaticFiles(directory="static"),
```

Ruff (0.14.8)

Undefined name `StaticFiles`

(F821)